index.html

```diff
@@ -7,6 +7,6 @@
   <title>Document</title>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy. Enjoy your stay!</h1>
```

Comment:
Could you remove 'Enjoy your Stay' string and make the 'Welcomo to Potionfy' string red?